README.md

```diff
@@ -30,6 +30,12 @@ module.exports = {
       //
       // Default arguments are `--typescript --target browser --mode normal`.
       extraArgs: "--no-typescript",
+
+      // If enabled, `watch` will always compile changes in `.rs` files, even
+      // if Webpack is not running in watch mode. Conversely, changing this to
+      // `false` will disable watch mode, even if Webpack is running in watch
+      // mode.
+      watch: true,
```

Comment:
What about renaming it to `forceWatch`  and keep the default aligned to Webpack's watch mode?

Reply:
That makes sense—just so we're clear, the functionality I implemented is:

- `watch === undefined` (i.e. not set) -> align to Webpack watch mode
- `watch === true` -> always watch
- `watch === false` -> never watch

Are you proposing only that I rename the option to `forceWatch`, or also that I change the flag so that it can only turn watch mode on, not off?

Reply:
Yes, what you implemented sounds looks to me. I was proposing to only rename the watch to forceWatch because it's actually forcing the mode. Absence of the key being undefined, the default would be aligned on webpack.

Also could you please comment it out in the documentation, user should know the exisintance of the option but probably not use it by default.

Reply:
Got it, thanks @xtuc!

I updated with these changes and made the documentation clearer.